views/resume/profile_resume.ejs

```diff
@@ -20,15 +20,33 @@
             $('select').material_select();
             $(".button-collapse").sideNav();
             $(".dropdown-button").dropdown();
-            document.getElementById("buts").onclick = function() {
-                document.getElementById("sub").submit();
-            }
+       
+            
+            $.get( "/getTags", function( data ) {
+                $('#technologies').val(data);
+                $('#technologies').material_select();
+            });
+
 
-            $('.chips').material_chip({
 
-            });
+
+            
 
         });
+
+
+        function update(){
+
```

Comment:
Weird spacing here; we should be as consistent as possible across all of the files. :)